content/docs/2.14/operate/setupscaler.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Setupscaler"
+weight = 500
+description = "Procedure to Setup a Scaler in KEDA"
++++
+
+## Prerequisites
+
+1. **Kubernetes Cluster**:
+   - Ensure you have a running Kubernetes cluster set up and accessible.
+   - If you don't have a cluster yet, follow the [official Kubernetes documentation](https://kubernetes.io/docs/setup/) to create a new cluster suitable for your environment (local machine, cloud provider, etc.).
+
+2. **KEDA Installation**:
+   - KEDA needs to be installed on your Kubernetes cluster before you can use it.
+   - Follow the [KEDA installation guide](https://keda.sh/docs/2.14/deploy/) carefully, including any prerequisites specific to your Kubernetes setup.
+   - The installation guide provides instructions for different installation methods (e.g., YAML, Helm charts, etc.). Choose the method that suits your needs.
+
+3. **kubectl**:
+   - The `kubectl` command-line tool is required to interact with your Kubernetes cluster.
+   - Follow the [official kubectl installation guide](https://kubernetes.io/docs/tasks/tools/#kubectl) to install `kubectl` on your operating system.
+   - Once installed, configure `kubectl` to communicate with your Kubernetes cluster by following the cluster-specific instructions provided by your Kubernetes setup.
+
+## Step 1: Identify the Scaler You Need
+
+KEDA supports various scalers that correspond to different event sources or triggers. Determining the right scaler is crucial for scaling your application based on the desired event source.
+
+1. Visit the [KEDA Scalers documentation](https://keda.sh/docs/2.14/scalers/) and browse through the list of available scalers.
+2. Identify the scaler that matches the event source you want to use for scaling your application. For example:
+   - If you want to scale based on incoming HTTP traffic, you would need the **HTTP scaler**.
```

Comment:
We don't have any HTTP Scaler atm, the closest one is the HTTP Add-on which is in beta status atm. I'm not 100% if we should include here (at least without a note).
@tomkerkhove  @zroubalik ?

Reply:
@JorTurFer so How should i proceed ? remove it or add a note

Reply:
@shubhusion add a note please